### PR TITLE
Update from auto-update/renku-notebooks-1.18.0

### PR DESCRIPTION
**Pull request by RenkuBot**
This is an automated pull request.

/deploy #persist

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -27,15 +28,28 @@ Internal Changes
...
   (`#1650 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1650>`_).
 
+- **Renku Notebooks:** parse old server options in request to start sessions
+  (`#1570 <https://github.com/SwissDataScienceCenter/renku-notebooks/issues/1570>`__)
+  (`8b3e5c0 <https://github.com/SwissDataScienceCenter/renku-notebooks/commit/8b3e5c091507446080fd468d84c4bd4b8d134b60>`__)
+
+- **Renku Notebooks:** properly recover LFS files from autosave branches
+  (`#1568 <https://github.com/SwissDataScienceCenter/renku-notebooks/issues/1568>`__)
+  (`8f34b09 <https://github.com/SwissDataScienceCenter/renku-notebooks/commit/8f34b09ab73913bfbba4acbe28b00c53ad576367>`__)
+
```
Comment (rokroskar) [inline CHANGELOG.rst:47]: should this be factored out and released as a bugfix for immediate deployment?

Comment (aledegano) [reply CHANGELOG.rst:47]: If possible I agree it would be better

Comment (olevski) [reply CHANGELOG.rst:47]: @rokroskar the single feature that comes from the notebook service is really simple and only adds a warning field when listing servers. So the whole notebooks release can be deployed immediately.

Comment (olevski) [reply CHANGELOG.rst:47]: I am not sure how I would isolate this from the KG changes which are already merged in the renku repo.

Comment (olevski) [reply CHANGELOG.rst:47]: I guess I can have a hotfix branch form which we can tag renku version 0.33.1 which will include only the notebooks release. Is that ok with you @rokroskar?

Comment (rokroskar) [reply CHANGELOG.rst:47]: yep - if that's not too much work I think that would be the way to go

**Comment by olevski** [comment]
@rokroskar @aledegano this updates the changelog for 0.33.1 and the requirements file. When you approve and merge this I will also publish 0.33.1 from a hotfix branch. But we can still use this branch/PR and merge this.

**Review by rokroskar — APPROVED** [review]

**Comment by olevski** [comment]
The draft release is here: https://github.com/SwissDataScienceCenter/renku/releases/tag/untagged-0fd8ec9d82ac9f612484

**Review by aledegano — DISMISSED** [review]

**Comment by olevski** [comment]
@rokroskar @aledegano, @jachro added his bug fixes to this PR instead of starting a new. We coordinated. We will combine all the bugfixes together in 0.33.1. He also modified the draft release in github for 0.33.1.

**Review by rokroskar — APPROVED** [review]

**Review by aledegano — APPROVED** [review]

**Pull request merged**